statsmodels/tsa/arima_process.py

```diff
@@ -108,13 +73,14 @@ def arma_generate_sample(ar, ma, nsample, sigma=1, distrvs=np.random.randn,
     >>> model.params
     array([ 0.79044189, -0.23140636,  0.70072904,  0.40608028])
     """
-    #TODO: unify with ArmaProcess method
-    eta = sigma * distrvs(nsample+burnin)
+    # TODO: unify with ArmaProcess method
+    eta = sigma * distrvs(nsample + burnin)
```

Comment:
(Also, how generally these kind of things actually work depends on the unit tests and example cases, which can often be just the ones that the developer thought about or usually used.)